lib/pkcs11h-core.c

```diff
@@ -1222,7 +1227,10 @@ pkcs11h_initializeProvider (
 #endif
 #endif
 #if !defined(_WIN32)
+	int loader_flags = RTLD_NOW | RTLD_LOCAL;
 	void *p;
+#else
+	DWORD loader_flags = 0;
```

Comment:
Can we initialize the default of the `loader_flags` at `pkcs11h_registerProvider` when we allocate the `_pkcs11h_provider_s`? This will simplify the code and ambiguity of the attribute.